internal/stage_file_search_2.go

```diff
@@ -0,0 +1,44 @@
+package internal
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/codecrafters-io/grep-tester/internal/test_cases"
+	"github.com/codecrafters-io/tester-utils/random"
+	"github.com/codecrafters-io/tester-utils/test_case_harness"
+)
+
+func testMultiLineFileSearch(stageHarness *test_case_harness.TestCaseHarness) error {
+	RelocateSystemGrep(stageHarness)
+
+	file_name := "fruits-" + randomFilePrefix() + ".txt"
+	fruits := append(random.RandomElementsFromArray(FRUITS, 3), "blueberry")
```

Comment:
Let's append two `berry` words here so that it's guaranteed that we test multiple matches - at the moment we might only test a single match